src/FakeItEasy/Creation/IFakeAndDummyManager.cs

```diff
@@ -13,7 +13,7 @@ internal interface IFakeAndDummyManager
         /// <param name="typeOfDummy">The type of dummy to create.</param>
         /// <returns>The created dummy.</returns>
         /// <exception cref="FakeItEasy.Core.FakeCreationException">The current IProxyGenerator is not able to generate a fake of the specified type and
-        /// the current IFakeObjectContainer does not contain the specified type.</exception>
+        /// the current IDummyValueCreationSession does not contain the specified type.</exception>

```

Comment:
Reasonable, especially since in the next method declaration on line 25 the comment on line 24 is exactly what you proposed. So this is a much more consistent comment anyway.